kubespawner/objects.py

```diff
@@ -80,6 +80,7 @@ def make_pod(
     lifecycle_hooks=None,
     init_containers=None,
     service_account=None,
+    automount_service_account_token=False,
```

Comment:
Default to None here assuming our passed configuration can be None, False, and True.